gradle.properties

```diff
@@ -1,4 +1,4 @@
-ARTIFACT_VERSION=1.1.1-onecloud
+version=1.0.0-SNAPSHOT
 ARTIFACT_NAME=flip-tables
```

Comment:
I added [this](https://github.com/Workiva/flip-tables/pull/4/files#diff-3d64996115d0ac3b74e36f57ce6b00d6b4982614c41b1ed39eab8ef43b2e9a2bR6-R10).  I don't think we really have a WK precedent for this but the Build system team doesn't think it will be a problem.